### DISCUSSION TITLE
Communication Print compatible?

### DISCUSSION THREAD

**Discussion by xbmcgotham** [arnemauer/Ducobox-ESPEasy-Plugin#117] [Q&A]
Hi, was hoping someone can let me know if its possible, and if so, how I can integrate a **DucoBox Energy Comfort** with **Communication Print** (https://www.duco.eu/nl/producten/mechanische-ventilatie/toebehoren/communication-print) into the **Home Assistant**. Its not clear to me if I can use **ESPEasy plugin** directly on the Communication Print. Hoping I can directly connect a RJ45 Ethernet cable to the Communication Print and than communicate with it within Home Assistant on the network.

Have a great weekend!

**Comment by arnemauer** [comment]
In dit topic staat volgens mij een oplossing daarvoor: https://gathering.tweakers.net/forum/list_messages/1724875/last

**Reply by xbmcgotham** [reply]
dankje voor de snelle reactie, denk alleen dat dit meer technisch is voor het opzetten van een verbinding als je geen communication print hebt. Ik ben opzoek naar een makkelijkere oplossing waarbij ik een RJ45 ethernet cable in de communication print plug en dan met home assistant de duco kan uitlezen en bedienen. Enige ervaring hiermee met opzetten?

**Reply by arnemauer** [reply]
Heb je wel gekeken in het topic?

In het topic staan diverse voorbeelden. Bijvoorbeeld: https://gathering.tweakers.net/forum/list_message/64751362#64751362
Zie ook de posts daarvoor en daarna.
